ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -166,32 +166,32 @@ tests:
       resources:
         requests:
           cpu: 100m
-- as: tests-extension-verify
+- as: tests-extension
   run_if_changed: ^(openshift/tests-extension/)
   steps:
     test:
-    - as: tests-extension-verify
+    - as: verify
       commands: |
         cd openshift/tests-extension
         make verify
       from: src
       resources:
         requests:
           cpu: 100m
-- as: tests-extension
-  steps:
-    test:
-    - as: tests-extension
+    - as: build
       commands: |
-        echo "Build binary olmv1-tests-ext"
         cd openshift/tests-extension
         make build
-        echo "Running olmv1-tests-ext with all tests"
-        ./bin/olmv1-tests-ext run-suite olmv1/all
```

Comment:
We need to replace it for the test done bellow with the ENV so that it will be executed as openshift/origin does